gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -71,14 +71,43 @@ fluid.defaults("gpii.flowManager", {
 fluid.defaults("gpii.flowManager.local", {
     // The flag to indicate whether to reset the system to the default settings on system startup
     resetAtStart: false,
-    // Distribute the grade to handle user logon requests.
-    // In all-in-local config, the handler is "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler".
-    // In untrusted config, the handler should be "gpii.lifecycleManager.untrusted.stateChangeHandler".
     distributeOptions: {
+        // Distribute the grade to handle user logon requests.
+        // In all-in-local config, the handler is "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler".
+        // In untrusted config, the handler should be "gpii.lifecycleManager.untrusted.stateChangeHandler".
         "lifecycleManager.loginRequestHandler": {
             record: "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler",
             target: "{that gpii.lifecycleManager.loginRequest}.options.gradeNames"
         },
+        // Both "defaultLifecycleInstructions" and "defaultSnapshot" structures are generated for performing
```

Comment:
This should really use the multi-line comment format - it is what it was invented for!